src/App.js

```diff
@@ -0,0 +1,29 @@
+import { Link, Route, Routes } from 'react-router-dom';
+import './App.css';
+import './styles/navbar.css';
+import Bookstore from './components/myBookstore';
+import Categories from './components/categories';
+
+function App() {
+  return (
+    <>
+      <nav className="navbar">
+        <h1 className="navbarTitle">Bookstore CMS</h1>
+        <ul className="navbarList">
+          <li>
+            <Link to="/">Books</Link>
+          </li>
+          <li>
+            <Link to="/Categories">Categories</Link>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [`OPTIONAL`] Great job so far initializing your project with components. However, I would like to suggest an improvement that will help make your code more component-based and also improve the **readability** and **maintainability** of your code. 
   -  I would like to suggest that you separate your **nav** into a different component called `Navbar` / `Nav`. You can use any name of your choice and import the newly created `Navbar` / `Nav` here in the `App.js`.